docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.5'
+
+volumes:
+  nginx-shared:
+
+services:
+  redis:
+    image: redis:5-alpine
+    ports:
+      - "6379:6379"
+
+  web:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+    env_file:
+      - .env
+    environment:
+      - ROCKET_DATABASES={service_cache={url="redis://redis"}}
```

Comment:
Shouldn't this be on the `.env` file?

Reply:
I moved it from the Rocket.toml to env for the dev configuration, but I will still overwrite it here as this is specific to this docker-compose config. Or would you recomend to create a separate env file for the docker setup (e.g. `.env.docker` for the docker-compose setup and `.env` is used for local setup)?

Reply:
I think is better, but for now it's ok this way